% to compile: \ll

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Document type      %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\documentclass[preprint,11pt,fleqn]{elsarticle}
\usepackage{nopageno} % no page numbers
\input{head/settings.tex}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Begin content      %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
%  \linenumbers
\small

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Title and author   %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frontmatter}

\title{Genomic and biochemical potential in transporter biofilter technology}
    \author[add1]{Placeholder title \corref{cor1}}
\author[add1]{Dylan Lawless\corref{cor1}}
	\ead{Dylan(dot)Lawless(at)epfl(dot)ch}
\cortext[cor1]{Addresses for correspondence}
\address[add1]{Global Health Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne, Lausanne, 1015, Switzerland}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Abstract           %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstract}
\setstretch{1.2}
Abstract
\end{abstract}
\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Formalities        %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %   \section*{Funding}
 %   \section*{Acknowledgements}
 %   \section*{Ethics statement}
 %   \section*{Key words}
    \section*{Abbreviations}
    \noindent 
   % ABC (ABC)

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Introduction       %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% \tableofcontents
% \listoffigures
%  \listoftables

\section{Introduction}
\noindent

\section{Screening complexity}
Whitelists, blacklists, and subject-specific definition. 


\section{Genomic screen}
Tolerated genomic sequences.
A benign microbe may acquire pathogenic sequences, plasmids.

\section{Peptide screen}

What is the number of unique proteins and peptide encoded by host genome.
What is the number of commensal organisms.


\section{Immonogenic tolerance filter}
The distance between foreign peptide and host peptides may indicate MHC presentation.
MHC presented unknown peptides may ilicite a flag - do not want to impede the normal host immune response. 
If flagged peptide producer can be identified it can be filtered as pathogen.
If not then send for further screen.

If peptides are too far from host cell or component then foreign material may be flagged.

How do we define commensal organisms. 
A subject specific immune profile should be recorded - HLA and somatic recombination matrix. 
We can predict the expected binding potential based on germline and somatic recombination of the HLA. 

\section{Decision tracing patter}
A fixed decision tree allows for learned patterns.


\section{Parasitic commensals}
Some organisms may be more parasitic than commensal, however, these apparent
pathogens may in-fact provide beneficial compoumds, hormones, or keep more damaging pathogens in check - commensal antibiotic. 
What criteria would save these organisms when all others would flag them for filtering?

Obligate parasites may need to be transferred to a synthetic host before assessing for extermination. 
The rules for sentience must be applied. 

\section{Host-pathogen co-evolution}
The benefit of host-pathogen interactions.
Genome to genome co-evolution.
Examples of a heterozygote advantage in which the heterozygous genotype has a higher relative fitness than either the homozygous dominant or homozygous recessive genotype. 
Consider things like CFTR, IFIH1. 
Think of a beneficial variants that causes immunodeficiency. 
Now think about a pathogen that might select in favour of a new host variant that is beneficial but without the pathogen pressure would not achieve selection. 
Is it ethical to remove pathogens that could have a unforseen benefit? 
Do not get into analytic philosophy, leave this to experts metaethics, normative ethics, and applied ethics that would rule on broader topics than biolfilters specifically.
We can apply statistical genomics to potentially give the uncertainty measure. 
This paper should only focus on \textit{can/how} and not on \textit{should}. 

A crude example with a pathogen and host variant is shown in the commented out latex: ...\\
% pathogen = infects 50 per 100 population\\
% pathogen = 40\% deaths (20 per 100)\\
% pathogen = 60\% survive (80 per 100)\\
% 
% p.R321L = 50 per 100 are carriers\\
% pathogen -->  p.R321L. 0\% deaths in carriers (10 per 100)\\
% 
% p.R321L = childhood hypomorphic phenotype. 10\% deaths  (5 per 100)\\
% p.R321L = adult selective advantage 30\% (15 per 100)\\ 
% 
% pathogen present = (20 per 100)\\
% pathogen and variants present = (10 per 100)\\
% variants present  = (5 per 100 death but  15 per 95 benefit later).\\
%
% Ignoring Hardy–Weinberg  in a crude exmple, we might see that a pathogen cases 20 deaths per 100. However, removing that pathogen would mean that a future host variant would not arise that results in 5 deaths and 15 benefits per 100. 
% Even if we knew this outcome we probably would remove the pathogen. 
% If we could predict the positive outcome then by default we would rule it neccessay to offer genetic modification the host in the first place rather than leave a preventable pathogen to do it over time. 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Conclusion         %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Conclusion}

\section*{Authorship Contributions}
%\noindent Dylan Lawless analysed data and conceived and wrote the manuscript, 
Filler content.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Conflict of Interest%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Conflict of Interest}
\noindent The authors declare no conflict of interest.

\clearpage

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% References         %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{\refname}
	\bibliographystyle{unsrtnat}
%	\bibliographystyle{plainnat}
	\bibliography{references}
\pagebreak 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% Supplemental       %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\beginsupplement
\section{Supplemental}
\label{sec:Supplemental_text}

% \begin{figure}[h]
%	\hspace*{0cm} \includegraphics[scale=0.62]{Supp_fig}
%	\caption{Caption filler}
%	\label{fig:Supp_fig}
%\end{figure}
%\clearpage

\end{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%% End of Document    %%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
